### PR TITLE
app-admin/clustershell: add 1.8.4 drop 1.8.3-r1

### PR DESCRIPTION
**Pull request by arkamar**
Version bump and old version removal, please see commit messages for more information.

### CODE REVIEW

**Comment by arkamar** [comment]
ping

File: app-admin/clustershell/clustershell-1.8.4.ebuild [hunk 1]
```diff
@@ -0,0 +1,71 @@
...
+)
+
+src_prepare() {
+	default
+
+	# remove test sets that require working ssh connection
+	rm tests/{CLIClush,TaskDistant*,TreeWorker}Test.py || die
+}
+
+distutils_enable_tests unittest
```
Comment (thesamesam) [inline app-admin/clustershell/clustershell-1.8.4.ebuild:50]: Please place above all phase definitions (so above src_prepare).

File: app-admin/clustershell/clustershell-1.8.4.ebuild [hunk 2]
```diff
@@ -0,0 +1,71 @@
...
+HOMEPAGE="https://github.com/cea-hpc/clustershell/"
+SRC_URI="
+	https://github.com/cea-hpc/clustershell/archive/v${PV}.tar.gz
+		-> ${P}.gh.tar.gz
+"
+
+LICENSE="LGPL-2.1+"
+SLOT="0"
+KEYWORDS="~amd64 ~x86"
+IUSE="test"
```
Comment (thesamesam) [inline app-admin/clustershell/clustershell-1.8.4.ebuild:22]: We can drop these two lines with d_e_t (which we're already using)

File: app-admin/clustershell/clustershell-1.8.4.ebuild [hunk 3]
```diff
@@ -0,0 +1,71 @@
...
+src_install() {
+	distutils-r1_src_install
+
+	doman doc/man/man*/*
+
+	mv "${ED}/usr/etc" "${ED}/etc" || die
+}
+
+pkg_postinst() {
+	einfo
```
Comment (thesamesam) [inline app-admin/clustershell/clustershell-1.8.4.ebuild:67]: Drop the no-content `einfo` commands, they're a bit cheeky/obnoxious

**Comment by arkamar** [comment]
I addressed all suggestions and rebased on top of master branch

File: app-admin/clustershell/clustershell-1.8.4.ebuild (hunk 2 above)
Comment (arkamar) [reply app-admin/clustershell/clustershell-1.8.4.ebuild]: uf, took me some time to substitute d_e_t correctly :)

**Comment by thesamesam** [comment]
Nice job and as usual, the commit message was a pleasure to read.

**Pull request closed without merging**